.goreleaser.yml

```
@@ -0,0 +1,53 @@
+# Visit https://goreleaser.com for documentation on how to customize this
+# behavior.
+before:
+  hooks:
+    # this is just an example and not a requirement for provider building/publishing
+    - go mod tidy
+builds:
+  - env:
+      # goreleaser does not work with CGO, it could also complicate
+      # usage by users in CI/CD systems like Terraform Cloud where
+      # they are unable to install libraries.
+      - CGO_ENABLED=0
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+      - -tags=all
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
```

Not 100% sure what this will do in our build.  We only have one link time variable setup and that's `main.versionNumber`, not sure what'll happen with `main.commit`.
```suggestion
      - '-s -w -X main.versionNumber={{.Version}}'
```